src/test/groovy/life/qbic/datamodel/dtos/business/AffiliationCategorySpec.groovy

```diff
@@ -0,0 +1,32 @@
+package life.qbic.datamodel.dtos.business
+
+import spock.lang.Specification
+
+
+/**
+ * Simple tests for the Affiliation Category Enum and its methods.
+ *
+ * @since 1.12.0
+ */
+class AffiliationCategorySpec extends Specification{
+
+    def "Established Enum properties can be accessed with the getValue and toString Methods "() {
+        when:
+        AffiliationCategory affiliationCategory = AffiliationCategory.INTERNAL
+
+        then:
+        affiliationCategory.getValue() == "internal"
+        affiliationCategory.toString() == "internal"
+    }
+
+    def "Trying to create an Enum with a non-specified Category throws MissingPropertyException"() {
+        when:
+        AffiliationCategory affiliationCategory = AffiliationCategory.UNKNOWN
+
+        then:
+        thrown(MissingPropertyException)
+    }
```

Comment:
Does it make sense to test for something that is not there? I would remove this
```suggestion

```

Reply:
Good Point! It probably is unnecessary to test an Enum anyway 👍